src/network/clientpackethandler.cpp

```diff
@@ -1376,9 +1376,13 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 				>> skybox.sky_color.indoors;
 		}
 
-		try {
+		if (pkt->getRemainingBytes() >= 4) {
 			*pkt >> skybox.body_orbit_tilt;
-		} catch (PacketError &e) {}
+		}
+
+		if (pkt->getRemainingBytes() >= 6) {
```

Comment:
`>= 0` would work too btw

Reply:
Yes. But isn't this (slightly) better?

Reply:
Not sure. If you write explicit sizes it's possible to get them wrong and introduce issues.

Reply:
Couldn't you just add your line into the try-catch block? (I think this is how we normally do it.)